bin/xbps-uhelper/xbps-uhelper.1

```diff
@@ -0,0 +1,131 @@
+.Dd July 12, 2020
+.Dt XBPS-UHELPER 1
+.Os
+.Sh NAME
+.Nm xbps-uhelper
+.Nd Utility to extract and compare package strings.
+.Sh SYNOPSIS
+.Nm
+.Op OPTIONS
+.Ar ACTION
+.Op ARGUMENTS
+.Sh DESCRIPTION
+.Nm
+provides a number of functions to extract and compare package strings.
+.Sh OPTIONS
+.Bl -tag -width -x
+.It Fl C Ar dir
+Path to xbps.conf file.
+.It Fl d
+Output debugging messages to stderr.
+.It Fl r Ar rootdir
+Use
+.Ar rootdir
+as rootdir.
+.It Fl V
+Prints the XBPS release version.
+.El
+.Sh ACTIONS
+.Bl -tag -width -x
+.It arch
+Return the XBPS native architecture.
+.It binpkgarch Ar binpkg
+Return architecture of package file
+.Ar binpkg .
+.It binpkgver Ar binpkg
+Return version of package file
+.Ar binpkg .
+.It cmpver Ar instver reqver
+Returns 255, 0 or 1 depending on whether the version of
+.Ar instver
+is less than, equal to, or greater than the version of
+.Ar reqver .
+.It fetch Ar URL
+Fetch a file from
+.Ar URL .
+.It getpkgdepname Ar pkgpattern
+Return the package name of dependency
+.Ar pkgpattern .
+.It getpkgdepversion Ar pkgpattern
+Return the package version of dependency
+.Ar pkgpattern .
+.It getpkgname Ar pkgver
+Return the name of package
+.Ar pkgver .
+.It getpkgrevision Ar pkgver
+Return the revision of package
+.Ar pkgver .
+.It getpkgversion Ar pkgver
+Return the version of package
+.Ar pkgver .
+.It getsystemdir
+Return the XBPS system directory.
+.It pkgmatch Ar pkg-version pkg-pattern
+Return 1 if
+.Ar pkg-version
+matches
+.Ar pkg-pattern ,
+0 otherwise.
+.It version Ar pkgname
+Return the version of installed package
+.Ar pkgname .
+.It real-version Ar pkgname
+Return the version of the real package to which virtual package
+.Ar pkgname
+resolves.
+.El
+.Sh EXAMPLES
+Return success if two packages are the same version, failure otherwise:
+.Pp
+.Dl $ xbps-uhelper cmpver 'foo-1.0_1' 'foo-2.1_1'
+.Pp
+.Dl $ xbps-uhelper cmpver 'foo-2.0' 'foo-2.1'
```

Comment:
I mean just version like `2.0 2.1`. In fact pkgver only work because they are passed directly into dewey algorithm, which, unlike xbps, accepts dashes in version.

Reply:
Oh, sorry for misunderstanding - PR updated.